R/connection.R

```diff
@@ -491,6 +491,40 @@ setMethod(
 ###############################################################################@
 ## Helpers ----
 
+.select_format <- function(choice){
```

Comment:
This function preserves "Arrow" as the default option but only errors if you actively choose "Arrow" and `{arrow}` is not installed. If you don't specify a format and arrow isn't installed, it will warn and fall back to TabSeparatedWithNamesAndTypes